ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/DeeplyNestedValueIT.scala

```diff
@@ -56,7 +57,7 @@ final class DeeplyNestedValueIT extends LedgerTestSuite {
     // The nesting of the key of a `ContractWithKey` is one more than the nat it contains
     val nKey = n - 1
 
-    def test[T](description: String)(
+    def test[T](description: String, errorCodeIfExpected: ErrorCode)(
```

Comment:
@remyhaemmerle-da Had to specialize the expected errors here based on where they are emitted: during preprocessing or interpretation. Can you double-check?

Reply:
:+1: